std/conv.d

```diff
@@ -289,6 +289,31 @@ template to(T)
     assert(to!int(to!float(-a)) == -a);
 }
 
+/**
+ * Conversion from string types to char types enforces the input
+ * to have a single code point, and then it converts said code point
+ * to its representation in a single code unit of the target type.
```

Comment:
That sounds needlessly complicated. I would simplify it to sth. like:

>  Conversion from string types to char types requires the input to consist of a single code point of the target type only. Otherwise a $(LREF ConvException) is thrown).

Reply:
No this is inaccurate. It must fit in a single code unit, not a code point.

It could be shortened. Just say “... code point, and that said code point fits into the target type”